.github/CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ Contributions are greatly appreciated! Some topics to contribute on:
 - Documenting parameter and return types
 - Naming unnamed natives appropriately
 - Adding example code to natives
-- Reviewing existing pull requests
+- Reviewing existing pull requests / issues
```

Comment:
I believe this file was mainly created as a guide on how to create PRs.

Reply:
Do you want me to remove `issues`? Would be great for people to know they can contribute answering the issues, no?

Reply:
Now that I think about it, you're correct. Since there's going to be more issues soon I believe.